pkg/helm/handlers/handlers.go

```diff
@@ -438,3 +441,28 @@ func (h *helmHandlers) HandleIndexFile(user *auth.User, w http.ResponseWriter, r
 
 	w.Write(out)
 }
+
+func (h *helmHandlers) HandleUninstallReleaseAsync(user *auth.User, w http.ResponseWriter, r *http.Request) {
+	params := r.URL.Query()
+	ns := params.Get("ns")
+	rel := params.Get("name")
+	version := params.Get("version")
+	conf := h.getActionConfigurations(h.ApiServerHost, ns, user.Token, &h.Transport)
+	coreClient, err := NewCoreClient(conf)
+	if err != nil {
+		serverutils.SendResponse(w, http.StatusBadRequest, serverutils.ApiError{Err: fmt.Sprintf("error forming core client: %s", err.Error())})
+		return
+	}
+	resp, err := h.uninstallReleaseAsync(rel, ns, version, conf, coreClient)
```

Comment:
I believe there is no benefit from returning the Secret as JSON here. We just need a confirmation:

```suggestion
	err := h.uninstallReleaseAsync(rel, ns, version, conf, coreClient)
```

Reply:
Agreed i will update the PR and the docs with `w.WriteHeader(http.StatusNoContent)`